be/tool/beck.c

```diff
@@ -400,14 +416,15 @@ static struct btype bt[] = {
 
 enum {
 	MAX_GEN    	 =     256,
+	MAX_SCAN_QUEUED	 = 1000000,
```

Comment:
Can we make it 10000000? This way we can ensure scanner can not be throttled by builder.
As per current speed of builder can process max 10000 actions and in node average action should be >= 128. builder will took 35.5 hours to complete this much node.While 1000000 can be processed by builder in 3.55 hours